src/main/java/lotto/domain/game/LottoNumber.java

```diff
@@ -54,12 +55,27 @@ public boolean equals(Object o) {
         if (this == o) return true;
         if (o == null || getClass() != o.getClass()) return false;
         LottoNumber that = (LottoNumber) o;
-        return Objects.equals(lottoNumbers, that.lottoNumbers);
+        return lottoNumber == that.lottoNumber;
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(lottoNumbers);
+        return Objects.hash(lottoNumber);
     }
 
+    @Override
+    public int compareTo(LottoNumber o) {
+        if (this.lottoNumber < o.lottoNumber) {
+            return -1;
+        }
+        if (this.lottoNumber > o.lottoNumber) {
+            return 1;
+        }
+        return 0;
```

Comment:
```suggestion
        return Integer.compare(this.lottoNumber, o.lottoNumber);
```
좀 더 간결하게 표현이 가능할 것 같습니다! 😊